Cypress tests fail

### Current behavior

I'm using Renovate for updating Node dependencies. It suggested using the latest version (11.0.2-canary.23) this morning, but the tests on the pull request fail with the following message:

```
The function exported by the plugins file threw an error.
We invoked the function exported by `[my_dir_path]/cypress/plugins/index.js`, but it threw an error.
Error: Cannot find module 'next/dist/next-server/server/config'
```

cypress/plugins/index.js:
```javascript
/// <reference types="cypress" />
/**
 * @type Cypress.PluginConfig
 */
module.exports = (on, config) => {
    const injectDevServer = require('@cypress/react/plugins/next')
    injectDevServer(on, config)

    return config
}
```

### Desired behavior

Tests should still be passing and not throwing an error.

### Test code to reproduce

Try using Cypress and Next.js with their latest version

### Cypress Version

8.0.0

### Other

_No response_

--- a/npm/react/plugins/next/findNextWebpackConfig.js
+++ b/npm/react/plugins/next/findNextWebpackConfig.js
@@ -1,10 +1,23 @@
 // @ts-check
 /// <reference types="next" />
 const debug = require('debug')('@cypress/react')
-const loadConfig = require('next/dist/next-server/server/config').default
 const getNextJsBaseWebpackConfig = require('next/dist/build/webpack-config').default
 
 async function getNextWebpackConfig (config) {
+  let loadConfig
+
+  try {
+    loadConfig = require('next/dist/next-server/server/config').default
+  } catch (e) {
+    if (e.code === 'MODULE_NOT_FOUND') {
+      // Starting from 11.0.2-canary.23, the server config file
+      // is not in the next-server folder anymore.
+      // @ts-ignore
+      loadConfig = require('next/dist/server/config').default
+    }
+
+    throw e
+  }
   const nextConfig = await loadConfig('development', config.projectRoot)
   const nextWebpackConfig = await getNextJsBaseWebpackConfig(
     config.projectRoot,

